fastlane/lib/fastlane/cli_tools_distributor.rb

```diff
@@ -23,7 +23,7 @@ def utf8_locale?
       def take_off
         before_import_time = Time.now
 
-        if !ENV["FASTLANE_DISABLE_ANIMATION"]
+        if ENV["FASTLANE_DISABLE_ANIMATION"].nil?
```

Comment:
This fixes a crash in rubocop. I would report it to them but in the meantime, we can keep it running with this easy fix.

https://gist.github.com/ainame/b90ba9fc159a0ff2ab8cb417ac9586c0

Reply:
How weird